### PR TITLE
[New] Add material-scrolltop@1.0.0 w/ git auto-update

### LINKED ISSUE #9387: [Request] Add material-scrolltop
**Library name:** material-scrolltop
**Git repository url:** https://github.com/bartholomej/material-scrolltop
**npm package url(optional):** 
**License(s):** MIT Licence
**Official homepage:** http://bartholomej.github.io/material-scrolltop
**Wanna say something? Leave message here:** Lightweight, Material Design inspired button for scroll-to-top of the page (jQuery plugin).
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #9387, cc @bartholomej, using web-minify-helper tool for minifing

Pull request for issue: #9387 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bartholomej/material-scrolltop
 * Official website (optional, not the repository): http://bartholomej.github.io/material-scrolltop/
 * NPM package url (optional): https://www.npmjs.com/package/material-scrolltop
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 6
   - Count of stars: 70
   - Count of forks: 16
 * NPM download stats (optional):
   - Downloads in the last day: 34
   - Downloads in the last week: 34
   - Downloads in the last month: 34

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/material-scrolltop/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bartholomej/material-scrolltop.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "material-scrolltop*.+(css|js)",
+          "images/*"
```
Comment (extend1994) [inline ajax/libs/material-scrolltop/package.json]: Let's modify this to `icons/*`

File: ajax/libs/material-scrolltop/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "sass"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bartholomej/material-scrolltop.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "material-scrolltop*.+(css|js)",
```
Comment (extend1994) [inline ajax/libs/material-scrolltop/package.json]: There might be `.map` files in the future. So let's update this to be `material-scrolltop*.+(css|js|map)`

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR message was updated. FYR

File: ajax/libs/material-scrolltop/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "material-scrolltop",
+  "filename": "material-scrolltop.min.js",
+  "version": "1.0.0",
+  "description": "Lightweight, Material Design inspired plugin for scrolling on top of the html page (with jQuery)",
+  "author": "Lukas Bartak <bart@bartweb.cz> (https://github.com/bartholomej)",
+  "repository": {
+    "url": "git+https://github.com/bartholomej/material-scrolltop.git",
+    "type": "git"
```
Comment (sufuf3) [inline ajax/libs/material-scrolltop/package.json]: Let's use 
```js
  "repository": {
    "type": "git"
    "url": "git+https://github.com/bartholomej/material-scrolltop.git",
  },
```
To keep the origin order.
Thanks

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Pull request merged**